.github/repository-settings.md

```diff
@@ -44,6 +42,10 @@ settings](https://github.com/open-telemetry/community/blob/main/docs/how-to-conf
     - `required-status-check`
     - `gradle-wrapper-validation`
   - Block force pushes: CHECKED
+  - Require code scanning results: CHECKED
+    - CodeQL
+      - Security alerts: High or higher
+      - Alerts: Errors
```

Comment:
I've gone ahead and updated this, let me know if you see anything weird as a result